auth0/src/main/java/com/auth0/android/provider/WebAuthProvider.java

```diff
@@ -158,16 +162,29 @@ public Builder withState(@NonNull String state) {
          * Give a scope for this request.
          *
          * @param scope to request.
+         * @return the current builder instance
          */
         public Builder withScope(@NonNull String scope) {
             this.scope = scope;
             return this;
         }
 
         /**
+         * Give a connection scope for this request.
+         *
+         * @param connectionScope to request.
+         * @return the current builder instance
+         */
+        public Builder withConnectionScope(@NonNull String connectionScope) {
```

Comment:
Let's use variable arguments or a list and join them when sending them to the server